photon-ml/src/main/scala/com/linkedin/photon/ml/avro/model/ModelProcessingUtils.scala

```diff
@@ -109,7 +118,8 @@ object ModelProcessingUtils {
         val Array(featureShardId) = IOUtils.readStringsFromHDFS(idInfoPath, configuration).toArray
 
         // Load the coefficients
-        val featureNameAndTermToIndexMap = featureShardIdToIndexMapLoader(featureShardId).indexMapForDriver()
+        val indexMapLoaders = featureShardIdToIndexMapLoader.flatMap(_.get(featureShardId))
+        val featureNameAndTermToIndexMap = indexMapLoaders.map(_.indexMapForDriver())
```

Comment:
It would be helpful for reader understanding to change the names of these vals to reflect that they are of class `Option[_]` (i.e. indexMapLoadersOption).